tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1007,6 +1021,12 @@ fdb/test_fdb_mac_learning.py::TestFdbMacLearning::testFdbMacLearning:
 #######################################
 #####            fib              #####
 #######################################
+fib/test_fib.py:
+  skip:
```

Comment:
xfail.

Reply:
This test will fail 100% of times on t1-isolated-d28u1, as there is only one upstream neighbor.

Reply:
hi Aharon, @Pterosaur is looking at this issue and with the recent fixes. What we are seeing is that this is not because of the one upstream neighbor, but some packet drops due to weird PTF behavior. Sometimes the packet could be stuck in kernel before handing off to the protocol stack, however, if we stop the PTF, the packet will show up in the tcpdump right away.

the test we are looking at is test_basic_fib and this is still under investigating. xfail should be better, unless you are referring to another test case.